src/components/ContentOpenWith/ContentOpenWith.js

```diff
@@ -368,6 +372,7 @@ class ContentOpenWith extends PureComponent<Props, State> {
                         <ExecuteForm
                             onSubmit={this.onExecuteFormSubmit}
                             executePostData={executePostData}
+                            windowRef={this.windowRef}
```

Comment:
Consider passing down only what the component needs, which is the window name. It doesn't seem to use the window object for anything else.